oleggerasimovich/parser.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import os
+
+from dotenv import load_dotenv
+import psycopg2
+
+load_dotenv()
+
+db = os.getenv('DATABASE')
+password = os.getenv('PASSWORD')
+host = os.getenv('HOST')
+user = os.getenv('USER')
+port = os.getenv('PORT')
+
+parser = argparse.ArgumentParser(description='Paste path to file')
```

Comment:
`description` parameter of `ArgumentParser` is actually a description of the whole script. Please, put meaningful description of the script itself.